src/esp/physics/bullet/BulletArticulatedObject.h

```diff
@@ -211,6 +211,13 @@ class BulletArticulatedObject : public ArticulatedObject {
    */
   std::vector<float> getJointPositions() override;
 
+  /**
+   * @brief Get the torques on each joint
+   *
+   * @return Array of torques on each joint
+   */
+  std::vector<float> getJointMotorTorques(double fixedTimeStep);
```

Comment:
Also add the param documentation here so the web doc generation will pick it up.

Reply:
@aclegg3 We would need to add these two new calls to the JS binding then as well.